docs/setup.md

```diff
@@ -27,7 +27,11 @@ cd matrix-hookshot
 yarn # or npm i
 ```
 
-Starting the bridge (after configuring it), is a matter of running `yarn start`.
+Starting the bridge (after configuring it), is a matter of setting the `NODE_ENV` environment variable to `production` or `development` and then running it:
```

Comment:
Should we explain what this does?

Reply:
Done in [2b85060](https://github.com/matrix-org/matrix-hookshot/pull/904/commits/2b85060fc150884ddc1b93a0399dacc1cd423ba6).